content/docs/details/entity-push-api/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Entity Push API
+publishedDate: '2024-02-16'
+description: How to use the Entity Push API
+---
+
+# Catalog your AWS accounts
+
+In this tutorial we are going to show you how to ingest your organization's AWS accounts into your Roadie catalog as Resource entities. For this we will use Roadie's Entity Push API you can check out the api docs [here](https://roadie.io/docs/api/catalog/) in the Roadie Provider section
```

Comment:
Maybe a sentence saying, "This tutorial will hopefully serve as an example around how you might apply this same pattern to other cloud providers or other resources that you cannot currently manage with the providers available with Roadie out of the box."